### DISCUSSION TITLE
Springboot 3 compatibility

### DISCUSSION THREAD

**Discussion by TomaszFenczyszyn** [Randgalt/record-builder#155] [Q&A]
Hi,

I've been trying to update my project from Springboot 2.6.7 to 3.0.0 and faced some compilation problems. Autogenerating record-builder classes doesn't seem to run before the compilation and all the references to record builders are being marked as missing class. I'm using Gradle and include soabase this way (which works perfectly fine for Springboot 2.6.7):
`annotationProcessor 'io.soabase.record-builder:record-builder-processor:36' 
    compileOnly 'io.soabase.record-builder:record-builder-core:36'`
I know that Springboot 3 introduced some changes to java records and for example lombok had to be upgraded to certain version for compatibility.
Does anyone know if record-builder library is compatible with Springboot 3? If not, is there any plan to cover it in the future?

Best regards,
 Tomasz Fenczyszyn

**Comment by Randgalt** [comment]
Can you post an example that shows the issue?

**Comment by frederikb** [comment]
Just as a heads up: we're using record-builder in combination with Spring Boot 3 and Maven without any issues.
